File: JavaScript/sort/BubbleSort.js
/**
 * 两两比较，将较大的放在后面。
 * 
 * @param A: an integer array
 * @return: 
 */
const sortIntegers = function (A) {
  if (A === null || A.length === 0) {
    return;
  }
  
  for (let i = 1; i < A.length - 1; i++) {
    for (let j = 0; j < i; j++) {
      if (A[i] > A[j]) {
        let temp = A[i];
        A[i] = A[j];
        A[j] = temp;
      }
    }
  }
}

File: Python/sort/BubbleSort.py
class Solution:
    """
    @ param A: an integer array
    @ return: nothing
    """
    def sortIntegers(self, A):
        # write your code here        

        for i in range(len(A)-1):
            for j in range(len(A)-i-1):
                if A[j] > A[j+1]:
                    temp = A[j]
                    A[j] = A[j+1]
                    A[j+1] = temp
        return A
        
